tasks/sanity-inspect-parent-image.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sanity-inspect-parent-image
+spec:
+  params:
+  - name: sourceImage
+    description: the fully qualified image name
+    default: quay.io/hongweiliu/skopeo-inspect-test:latest
+    type: string
+  steps:
+  - name: inspect-image
+    image: quay.io/redhat-appstudio/hacbs-test:latest
+    script: |
+      # 'org.opencontainers.image.base.name' exists for OCI format image only but does not exist in docker format image
+      # So we should check if  when the image is built by docker format image
```

Comment:
It doesn't look like the code to check the docker format image is in place? If I remember correctly, in the first day of the M4 F2F someone suggested we could use this to determine the base image

`skopeo inspect --config --no-tags docker://quay.io/mkovarik/spring-petclinic:dc80a7f | grep FROM`

Although using that example it shows that it was using an image from a secure registry or probably redhat access catalog.

`image-registry.openshift-image-registry.svc:5000/openshift/java:openjdk-11-ubi8`

That being said that example petclinic app image is using label `io.openshift.s2i.build.image` so I wonder if this another label we need to look for to determine the base image?

Reply:
`skopeo inspect --config --no-tags docker://quay.io/mkovarik/spring-petclinic:dc80a7f | grep FROM` is not deriving from the `From` statement inf Dockerfile, it is reading from comment when building, so it is not real `From`.
```
 % skopeo inspect --config --no-tags docker://quay.io/mkovarik/spring-petclinic:dc80a7f | grep FROM
            "comment": "FROM image-registry.openshift-image-registry.svc:5000/openshift/java:openjdk-11-ubi8",
```